package.json

```diff
@@ -1,27 +1,40 @@
 {
-  "name": "task1",
+  "name": "single_page_application",
   "version": "1.0.0",
-  "description": "Task 1 in React mentoring",
+  "description": "React mentoring",
   "main": "index.js",
   "scripts": {
-    "start server": "node server/server.js",
-    "build": "webpack --mode production"
+    "start": "webpack-dev-server --open --hot --config webpack.dev.js",
+    "build": "webpack -p --config webpack.prod.js",
+    "test": "jest"
   },
   "author": "Artur Liup",
   "license": "MIT",
   "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "express": "latest",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
```

Comment:
Look, all dependencies you import and use in runtime files - files contains code that executes in browser - should be listed in dependencies.
All other staff - everything you need to deploy, build, testing - devDependencies.

Reply:
done